### DISCUSSION TITLE
Getting errors when updating explorer patcher

### DISCUSSION THREAD

**Discussion by HarisHashim** [valinet/ExplorerPatcher#3882] [General]
Error as per image below:

![image](https://github.com/user-attachments/assets/11f51893-3491-4ccd-bec3-4f0a46f556a4)

Edition	Windows 11 Home Single Language
Version	23H2
OS build	22631.4317

EP verision

![image](https://github.com/user-attachments/assets/f1a02694-eaca-4bdb-a32b-85aaed8617c2)

Tried starting EP Properties window as admin but receive same error

* UPDATE
- Browse to  ${user}\AppData\Roaming\ExplorerPatccher and found a file named

"Update for ExplorerPatcher from https꞉∕∕github.com∕valinet∕ExplorerPatcher∕releases∕latest∕download∕ep_setup.exe"

Try to execute this file and get this error

![image](https://github.com/user-attachments/assets/4150d05b-e082-4664-9b30-dad5ad35252e)

**Comment by HarisHashim** [comment]
Disable Virus & threat protection settings => Real-Time protection and attempt update again. That solve this and and successfully upgrade to 22621.4317.67.1.

**Reply by pyrates999** [reply]
You can also add the path that the ep_setup.exe is being downloaded to and then it should be able to update.

**Reply by pyrates999** [reply]
Here's the command in powershell to run with elevated privileges:

`Add-MpPreference -ExclusionPath "$env:APPDATA\ExplorerPatcher\Update For ExplorerPatcher\"`
